src/Cache/FileCache.php

```diff
@@ -29,7 +29,8 @@ class FileCache implements CacheInterface
      */
     private static function getGherkinVersionHash(): string
     {
-        $version = InstalledVersions::getVersion('behat/gherkin');
+        $version = InstalledVersions::getVersion('behat/gherkin')
+            ?? throw new CacheException('Cannot detect behat/gherkin package version');
```

Comment:
I cannot reasonable cover this with a test (it relies on a lot of static stuff), any ideas? Shall we close an eye on this (leave it as is or add codeCoverageIgnore)?

Reply:
I also thought about it, but suspected it might not be desired. I'm fine doing it like that though.

Reply:
I'm using 'unknown'. Please reopen if you change your mind.